en/people/ludovico-poggi.md

```diff
@@ -0,0 +1,26 @@
+is_new: false
+twitter_user: ludo_poggi
+medium_user:
+linkedin_url: https://www.linkedin.com/in/ludovico-poggi-212a738/
+ref: ludovico-poggi
+parent_ref: team
+start_date: February 1st, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 67.000
+approved_by_court: May 3rd, 2017
+---
+I started my career at lavoce.info, an italian think tank and economic policy watch dog founded in 2002 by Tito Boeri. During this experience I learnt how to differentiate comments and facts, and used the latter to develop a solid assessment of the process of policy making, independently from the political parties that generated it. In this laboratory of ideas, where independence is of utmost importance, I managed to meet tight deadlines and covered different areas of the communication universe: press office, editing, fundraising, social media management and organization of events. I developed a learning-by-doing approach relying extensively on a wide range of instruments: internet, Twitter,(io li toglierei... in fin dei conti siamo nel 2017) a weekly newsletter, crowdfunding events and the community management.
+
+Whilst working at lavoce.info, I have been a free-lance journalist: as an elections reporter in Greece and Spain, an editor in chief of a web-journal focused on gender economy and as a project manager for a proposal that evaluated the use of internet among enterprises located in Lombardy. I have also worked as a consultant for the development of a Public Administration’s websites and contributed to edit a book on economic policy proposals to the national government.
+
+This experience, as well as the friendship that bore with Tommaso Nannicini, the curator of the book, would prompt me to move from Milan to Rome in February 2016. In the beautiful surroundings of the eternal city, I worked as a Director of Communications for Tommaso, then Undersecretary of State to the Prime Minister for Economic, Social and Research Policy. I soon realized the difference that lies between assessing government economic policy as an independent journalist and contributing to shape the same policies as an Undersecretary cabinet member.
+
+After few months, thanks to one of these policies (18app, a platform that provides a cultural bonus to the young adults turning 18), I met Diego Piacentini, the Government Commissioner for the Digital Agenda. Since the first meeting I noticed his analytical and managerial approach to problems and his ability to move forward incrementally and without any pre-conceived ideology. I am still learning this useful method and one day I would hope the entire public administration could change its paradigm and adopt the same modus operandi.
+
+In December 2016, in the aftermath of the Renzi’s government resignation, my contract expired and I discovered that Piacentini’s Digital Team was hiring: among the vacancies, I applied for the position in the Communications and institutional relations department.
```

Comment:
I'd say "I applied for the position in the Communications and institutional relations department." -> "I applied for the position in the Communications and institutional affairs department." to be aligned with the title chosen.